inst/htmljs/ldavis.js

```diff
@@ -95,6 +95,12 @@ LDAvis = function(to_select, json_file) {
     // see if vis is being run as part of a shiny app
     // (code adapted from https://github.com/ramnathv/htmlwidgets/blob/master/inst/www/htmlwidgets.js#L15)
     var inShinyMode = typeof(window.Shiny) !== "undefined" && !!window.Shiny.outputBindings;
+    
+    if (inShinyMode) {
+      var outputId = to_select.substring(1, to_select.length);
+      var topicClicked = outputId + "_topic_click";
+      var termClicked = outputId + "_term_click";
+    }
```

Comment:
Nice, thanks! I'd prefer more descriptive variable names here, say `shinyClickedTopic`/`shinyClickedTerm`

Once that's done, this looks good to me!

Reply:
done!